### DISCUSSION TITLE
en/installing-vpn-server-on-router-openwrt-wireguard/

### DISCUSSION THREAD

**Discussion by giscus** [idarek/dariusz.wieckiewicz.org#123] [General]
# en/installing-vpn-server-on-router-openwrt-wireguard/

Free world, free thoughts, free words.

https://dariusz.wieckiewicz.org/en/installing-vpn-server-on-router-openwrt-wireguard/

**Comment by kostas-77** [comment]
Very analytical and very helpful. Thank you.
